zh_cn/deco.gitterpane.json

```diff
@@ -1,20 +0,0 @@
-			"8": [{"name":"futurepack:metal_gitter_block", "meta":0},{"name":"futurepack:color_gitter", "meta":0},{"name":"futurepack:color_gitter", "meta":1},{"name":"futurepack:color_gitter", "meta":2},{"name":"futurepack:color_gitter", "meta":3},{"name":"futurepack:color_gitter", "meta":4},{"name":"futurepack:color_gitter", "meta":5},{"name":"futurepack:color_gitter", "meta":6},{"name":"futurepack:color_gitter", "meta":7},{"name":"futurepack:color_gitter", "meta":8},{"name":"futurepack:color_gitter", "meta":9},{"name":"futurepack:color_gitter", "meta":10},{"name":"futurepack:color_gitter", "meta":11},{"name":"futurepack:color_gitter", "meta":12},{"name":"futurepack:color_gitter", "meta":13},{"name":"futurepack:color_gitter", "meta":14},{"name":"futurepack:color_gitter", "meta":15}],
-			"9": [{"name":"futurepack:metal_gitter_block", "meta":0},{"name":"futurepack:color_gitter", "meta":0},{"name":"futurepack:color_gitter", "meta":1},{"name":"futurepack:color_gitter", "meta":2},{"name":"futurepack:color_gitter", "meta":3},{"name":"futurepack:color_gitter", "meta":4},{"name":"futurepack:color_gitter", "meta":5},{"name":"futurepack:color_gitter", "meta":6},{"name":"futurepack:color_gitter", "meta":7},{"name":"futurepack:color_gitter", "meta":8},{"name":"futurepack:color_gitter", "meta":9},{"name":"futurepack:color_gitter", "meta":10},{"name":"futurepack:color_gitter", "meta":11},{"name":"futurepack:color_gitter", "meta":12},{"name":"futurepack:color_gitter", "meta":13},{"name":"futurepack:color_gitter", "meta":14},{"name":"futurepack:color_gitter", "meta":15}],
-			"out": [{"name":"futurepack:metal_gitter_pane", "meta":0, "size":6},{"name":"futurepack:color_gitter_pane", "meta":0, "size":6},{"name":"futurepack:color_gitter_pane", "meta":1, "size":6},{"name":"futurepack:color_gitter_pane", "meta":2, "size":6},{"name":"futurepack:color_gitter_pane", "meta":3, "size":6},{"name":"futurepack:color_gitter_pane", "meta":4, "size":6},{"name":"futurepack:color_gitter_pane", "meta":5, "size":6},{"name":"futurepack:color_gitter_pane", "meta":6, "size":6},{"name":"futurepack:color_gitter_pane", "meta":7, "size":6},{"name":"futurepack:color_gitter_pane", "meta":8, "size":6},{"name":"futurepack:color_gitter_pane", "meta":9, "size":6},{"name":"futurepack:color_gitter_pane", "meta":10, "size":6},{"name":"futurepack:color_gitter_pane", "meta":11, "size":6},{"name":"futurepack:color_gitter_pane", "meta":12, "size":6},{"name":"futurepack:color_gitter_pane", "meta":13, "size":6},{"name":"futurepack:color_gitter_pane", "meta":14, "size":6},{"name":"futurepack:color_gitter_pane", "meta":15, "size":6}]
-		}
-	},
-	"信息：可使用喷枪进行染色"
-]
```

Comment:
why was this file deleted?

Reply:
They have no practical effect.
There is no corresponding file in folder [en_us](https://github.com/Wugand/FuturePack-Language/tree/master/en_us).
This recipe is repeated with [deco.gitter.json](https://github.com/Wugand/FuturePack-Language/blob/master/zh_cn/deco.gitter.json#L5)。